src/ucp/core/ucp_ep.c

```diff
@@ -116,7 +117,9 @@ ucs_status_t ucp_ep_new(ucp_worker_h worker, const char *peer_name,
     }
 
 #if ENABLE_DEBUG_DATA
-    ucs_snprintf_zero(ep->peer_name, UCP_WORKER_NAME_MAX, "%s", peer_name);
+    ucs_snprintf_zero(ep->peer_name, ucs_max(UCP_WORKER_NAME_MAX,
```

Comment:
i think it's better to define the array size as UCP_WORKER_NAME_MAX, unrelated to sockaddr string len